### ISSUE TITLE
test_cases.csv is ambiguous?

### ISSUE THREAD

**Issue by matt1795** [multiformats/multihash#123]
Hey, just a heads up that test_cases.csv is hashing the input as a string and not as a binary. Is this intended? From my perspective, it looks like the input is intended to be base16 encoded binary.

**Comment by Stebalien** [comment]
I agree. At this point, "known" is probably more accurate than "intended".

If you'd like to change this, go for it. If you do that, please update https://github.com/multiformats/go-multihash/blob/249ead2008065c476a2ee45e8e75e8b85d846a72/spec_test.go#L70 to match.
